lib/tapjoy/ldap/key/remove.rb

```diff
@@ -0,0 +1,92 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Remove a user key from user profile
+      class Remove
+        # Remove key from LDAP
+        def remove
+          keys  # Get keys first
+          Tapjoy::LDAP::Key.verify_user(opts[:user], results)
+
+          confirm unless opts[:force]
+          Tapjoy::LDAP::client.replace_attribute(
+            @user_dn, :sshPublicKey, keep_keys)
+        end
+
+        private
+        def opts
+          @opts ||= Trollop::options do
+            # Set help message
+            usage 'key remove [options]'
+            synopsis "\nThis command is for removing a user's SSH key(s)"
+
+            opt :user, 'Specify username to delete key from', type: :string,
+                required: true
+            opt :filename, 'File to load key deletion list from', type: :string
+            opt :force, 'Force delete', short: '-F'
+          end
+        end
+
+        def keys
+          @keys ||= Tapjoy::LDAP::Key.get_keys_from_commandline(opts[:filename])
```

Comment:
Oh wait, I see how you are doing this we probably want to do have this adopt different behavior, if you provide a --key string we try and delete that, otherwise we bring up a dialogue where you can choose which keys you want to delete.